pkg/os/exec.go

```diff
@@ -20,7 +20,6 @@ func run(command string, args []string, env map[string]string) (string, string,
 	stdErr := new(bytes.Buffer)
 	cmd.Stdout = stdOut
 	cmd.Stderr = stdErr
-	logging.Debugf("Running '%s %s'", command, strings.Join(args, " "))
 	err := cmd.Run()
 	if err != nil {
 		logging.Debugf("Command failed: %v", err)
```

Comment:
Wondering if `oc` will print the failed command/args and leak the pull-secret?

Reply:
My assumtion is oc command mostly fail either the apiserver is not up or the resource is not valid and both the cases I think we are safe.